src/App.css

```diff
@@ -1,33 +1,35 @@
 .App {
-  display: flex;
-  flex-direction: column;
-  width: 100vw;
-  height: 100vh;
-  overflow-x: hidden;
-  background-color: #FAFAFA;
+	display: flex;
+	flex-direction: column;
+	width: 100vw;
+	height: 100vh;
+	overflow-x: hidden;
+	background-color: #fafafa;
 }
 .App .main {
-  flex: 1;
-  display: flex;
-  flex-direction: row;
+	flex: 1;
+	display: flex;
+	flex-direction: row;
 }
 .AppBar {
-  position: fixed !important;
+	position: fixed !important;
+	-moz-user-select: none; /* Firefox */
```

Comment:
There is no need to support all browsers, as this is an electron app and `user-select: none;` will suffice.